src/OrchardCore/OrchardCore.Abstractions/Shell/ShellSettings.cs

```diff
@@ -75,8 +74,5 @@ public TenantState State
         }
 
         public Task EnsureConfigurationAsync() => _configuration.EnsureConfigurationAsync();
-
-        public bool IsDefaultShell()
-            => String.Equals(Name, ShellHelper.DefaultShellName, StringComparison.OrdinalIgnoreCase);
```

Comment:
Why is it better not to have this? Even if we do an exact string comparison, I'd prefer to write `shellSettings.IsDefaultShell()` over `shellSettings.Name == ShellHelper.DefaultShellName` each time (quicker to write, shorter, a bit clearer semantics). I don't really see why moving it to an extension method would be better ([this](https://rules.sonarsource.com/csharp/RSPEC-4226) is a counter-argument too).

(From [this comment of yours](https://github.com/OrchardCMS/OrchardCore/pull/11435#issuecomment-1201890258) I understand the argument for not needing `OrdinalIgnoreCase`.)

Reply:
> I understand the argument for not needing OrdinalIgnoreCase

Cool, so many discussions (but maybe necessary) to agree on this main point.

> Why is it better not to have this?

Here I didn't use `IsDefaultShell()` intentionally to clearly show what I think is better to do in a given context, e.g checking the name on an injected ShellSettings, people mix the way we search by name (now case insensitive oob) with what we have to check on the name of a given instance.

For sure we can use an helper if it is case sensitive, but was not sure about the name. For me it depends on what can be done, for a shellHost ext `IsDefaultShellSettings()`, for a string ext `IsDefaultShellName()`, for a shellSettings ext `HasDefaultShellName()` to only describe what it does.

So I'm fine to use `IsDefaultShell()` if it is case sensitive, maybe `HasDefaultShellName()` ;)

> I don't really see why moving it to an extension method would be better ([this](https://rules.sonarsource.com/csharp/RSPEC-4226) is a counter-argument too).

Hmm, maybe I suggested this when saying why not also having e.g. `HasRunningState()`, `HasInvalidState()` and so on, to group them in a separate file, but no closed opinion here.